cluster/calcium/node.go

```diff
@@ -75,14 +76,25 @@ func (c *Calcium) SetNode(ctx context.Context, opts *types.SetNodeOptions) (*typ
 				return logger.Err(ctx, errors.WithStack(err))
 			}
 			for _, workload := range workloads {
+				appname, entrypoint, _, err := utils.ParseWorkloadName(workload.Name)
+				if err != nil {
+					log.Errorf(ctx, "[SetNodeAvailable] Set workload %s on node %s inactive failed %v", workload.ID, opts.Nodename, err)
+					continue
+				}
+
 				if workload.StatusMeta == nil {
 					workload.StatusMeta = &types.StatusMeta{ID: workload.ID}
 				}
 				workload.StatusMeta.Running = false
 				workload.StatusMeta.Healthy = false
 
+				// Set these attributes to set workload status
+				workload.StatusMeta.Appname = appname
+				workload.StatusMeta.Nodename = workload.Nodename
+				workload.StatusMeta.Entrypoint = entrypoint
```

Comment:
We don't have agent's reporting information here, so we need to parse `workload.Name`